lms/djangoapps/certificates/models.py

```diff
@@ -870,6 +870,14 @@ class CertificateGenerationCourseSetting(TimeStampedModel):
             u"certificate template."
         )
     )
+    include_hours_of_effort = models.BooleanField(
```

Comment:
Should we set a default? And can this be null?

Reply:
I set the default to True so that, unless it is explicitly not wanted, spanish course certificates would have the data to render Hours of Effort.
I could add a not null constraint, but the other fields in the CertificateGenerationCourseSetting model don't have that constraint, and I think it would be reasonable to treat null values the same as False values for this field.

Reply:
I think it might be better to default to False, since that's the current status quo. We probably don't want this to randomly start showing up in certificates for which we haven't explicitly turned it on.

All Spanish courses that are going to have translated certificates will need a configuration row anyway in order to enable translated certificates. As we're adding those rows, we can make sure to turn on 'hours of effort' when required.

Reply:
I would rather do one of the following:
No default and treat null (aka lack of a choice) as false since this is a new feature.  Then we can eventually turn it on for everyone with a code change treating the lack of a choice as true.

OR

Default to false, add a null constraint, and when we want to release this feature generally we create a migration to update the default to true.



My preference is the first because it shows intent more clearly (True, False, Non-choice)

Reply:
@bderusha just to clarify, the first option would have neither a default nor a null constraint, yes?